### PR TITLE
core term links and capitalization - events reference

### PR DESCRIPTION
**Pull request by DJSanti**
Links and capitalization fixes for reference pages.

### CODE REVIEW

File: docs/references/events.md [hunk 1]
```diff
@@ -25,18 +25,18 @@ This is always the first Event in a Workflow Execution History. It indicates tha
...
 | last_completion_result                                                       | Information from the previously completed Task, if applicable.                                                                                  |
 | original_execution_run_id                                                    | The [Run Id](/concepts/what-is-a-run-id) of the original Workflow started.                                                                      |
 | identity                                                                     | Id of the Worker that requested the start of this Workflow.                                                                                     |
-| first_execution_run_id                                                       | The first run Id, along the chain of ContinueAsNew and Reset.                                                                                   |
+| first_execution_run_id                                                       | The first Run Id, along the chain of ContinueAsNew and Reset.                                                                                   |
```
Comment (djmagee) [inline docs/references/events.md:34]:
```suggestion
| first_execution_run_id                                                       | The first Run Id, along the chain of Continue-As-New and Reset.                                                                                 |
```

File: docs/references/events.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ This indicates that the Workflow Execution has successfully completed. The Event
 | -------------------------------- | ----------------------------------------------------------------------------------------------- |
 | result                           | Serialized result of completed Workflow.                                                        |
 | workflow_task_completed_event_id | The Id of the [WorkflowTaskCompleted](#workflowtaskcompleted) that the Event was reported with. |
-| new_execution_run_id             | The [Run Id](/concepts/what-is-a-run-id) of the new Workflow started by cron.                   |
+| new_execution_run_id             | The [Run Id](/concepts/what-is-a-run-id) of the new Workflow started by Cron.                   |
```
Comment (djmagee) [inline docs/references/events.md:53]: Despite the terms Cron Job and Cron Schedule, when we refer to cron by itself, the styling is lowercase—I think. @flossypurse, concur or disagree?

File: docs/references/events.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ This Event indicates that the Workflow Execution has unsuccessfully completed an
 | failure                          | Serialized result of a Workflow failure.                                                                                          |
 | retry_state                      | The reason provided for whether the Task should or shouldn't be retried.                                                          |
 | workflow_task_completed_event_id | The [Run Id](/concepts/what-is-a-run-id) of the [WorkflowTaskCompleted](#workflowtaskcompleted) that the Event was reported with. |
-| new_execution_run_id             | The [Run Id](/concepts/what-is-a-run-id) of the new Workflow started by cron or Retry.                                            |
+| new_execution_run_id             | The [Run Id](/concepts/what-is-a-run-id) of the new Workflow started by Cron or Retry.                                            |
```
Comment (djmagee) [inline docs/references/events.md:64]: See discussion about cron on line 53.

Comment (DJSanti) [reply docs/references/events.md:64]: I did this to get organized. I'm still working on it

Comment (djmagee) [reply docs/references/events.md:64]: Ah. I jumped the gun. Back to the starting line with me.

Comment (DJSanti) [reply docs/references/events.md]: okay, all good now

File: docs/references/events.md [hunk 4]
```diff
@@ -8,105 +8,105 @@ tags:
...
+| continued_failure                                                            | Serialized result of a failure.                                                                                                                          |
+| last_completion_result                                                       | Information from the previously completed [Task](/concepts/what-is-a-task), if applicable.                                                               |
+| original_execution_run_id                                                    | The [Run Id](/concepts/what-is-a-run-id) of the original Workflow started.                                                                               |
+| identity                                                                     | Id of the [Worker](/concepts/what-is-a-worker) that requested the start of this Workflow.                                                                |
+| first_execution_run_id                                                       | The first Run Id, along the chain of [ContinueAsNew](/concepts/what-is-continue-as-new) and Reset.                                                       |
```
Comment (flossypurse) [inline docs/references/events.md:34]:
```suggestion
| first_execution_run_id                                                       | The first [Run Id](/concepts/what-is-a-run-id), along the chain of [Continue-As-New](/concepts/what-is-continue-as-new) Runs and Reset.                                                       |
```

File: docs/references/events.md [hunk 5]
```diff
@@ -8,105 +8,105 @@ tags:
...
+| [retry_policy](/concepts/what-is-a-retry-policy)                             | The amount of retries as determined by the service's dynamic configuration. Retries will happen until 'schedule_to_close_timeout' is reached.            |
+| attempt                                                                      | The number of attempts that have been made to complete this Task.                                                                                        |
+| workflow_execution_expiration_time                                           | The absolute time at which the Workflow will be [timed out](/concepts/what-is-a-workflow-execution-timeout).                                             |
```
Comment (flossypurse) [inline docs/references/events.md:37]:
```suggestion
| workflow_execution_expiration_time                                           | The absolute time at which the Workflow Execution will [time out](/concepts/what-is-a-workflow-execution-timeout).                                             |
```

File: docs/references/events.md [hunk 6]
```diff
@@ -8,105 +8,105 @@ tags:
...
+This indicates that the [Workflow Execution](/concepts/what-is-a-workflow-execution) has successfully completed. The [Event](/concepts/what-is-an-event) contains Workflow Execution results.
 
 | Field                            | Description                                                                                     |
 | -------------------------------- | ----------------------------------------------------------------------------------------------- |
-| result                           | Serialized result of completed Workflow.                                                        |
+| result                           | Serialized result of completed [Workflow](/concepts/what-is-a-workflow).                        |
 | workflow_task_completed_event_id | The Id of the [WorkflowTaskCompleted](#workflowtaskcompleted) that the Event was reported with. |
-| new_execution_run_id             | The [Run Id](/concepts/what-is-a-run-id) of the new Workflow started by cron.                   |
+| new_execution_run_id             | The [Run Id](/concepts/what-is-a-run-id) of the new Workflow started by Cron.                   |
```
Comment (flossypurse) [inline docs/references/events.md:53]:
```suggestion
| new_execution_run_id             | The [Run Id](/concepts/what-is-a-run-id) of the new Workflow Execution started as a result of a [Cron Schedule](/concepts/what-is-a-temporal-cron-job).                   |
```

**Comment by flossypurse** [comment]
Overall, let's de-link the field names - for example:
```
NO -> [workflow_execution_timeout](/concepts/what-is-a-workflow-execution-timeout) 
Yes -> workflow_execution_timeout
```

File: docs/references/events.md (hunk 2 above)
Comment (flossypurse) [reply docs/references/events.md:53]: Sure, but I don't know of when we would use it generically - 
It is either Temporal Cron Job, or Cron Schedule I think.

**Review by djmagee — APPROVED** [review]

**Pull request merged**